iroh-blobs/src/util/progress.rs

```diff
@@ -518,6 +518,98 @@ impl<T: Send + Sync + 'static> ProgressSender for FlumeProgressSender<T> {
     }
 }
 
+/// A progress sender that uses a flume channel.
```

Comment:
```suggestion
/// A progress sender that uses an async channel.
```

Reply:
@rklaehn I think you are still missing this and another quick doc update suggestion :v: